frontend/js/src/playlists/components/PlaylistMenu.tsx

```diff
@@ -224,114 +224,108 @@ function PlaylistMenu({
   );
   const showAppleMusicExportButton = appleAuth;
   return (
-    <ul
+    <div
```

Comment:
Ahhhhh, not requiring a specific html element structure anymore?
That does put a smile on my face...